modelbaker/iliwrapper/ili2dbutils.py

```diff
@@ -347,7 +347,7 @@ def error_string(self):
         if self.java_version:
             return QCoreApplication.translate(
                 "ili2dbutils",
-                'Wrong java version found. Qgis Model Baker requires at least java version 8. Please <a href="{0}">install Java</a> and or <a href="{1}">configure a custom java path</a>.<br/><br/>Java Version:<br/>{2}',
+                'Wrong java version found. modelbaker requires at least java version 8. Please <a href="{0}">install Java</a> and or <a href="{1}">configure a custom java path</a>.<br/><br/>Java Version:<br/>{2}',
```

Comment:
Is that change intentional? Should not be just "Model Baker" as it is everywhere in the GUI?

Reply:
I took modelbaker to make a difference between the backend "modelbaker" and the frontend "QGIS Model Baker", but since this message appears in the GUI I agree that "Model Baker" would be better. I'll change that.